src/utils/actionTypes.js

```diff
@@ -4,8 +4,9 @@
  * If the current state is undefined, you must return the initial state.
  * Do not reference these action types directly in your code.
  */
-var ActionTypes = {
-  INIT: '@@redux/INIT'
+const ActionTypes = {
+  INIT: '@@redux/INIT' + Math.random().toString(36).substring(7).split('').join('.'),
+  REPLACE: '@@redux/REPLACE' + Math.random().toString(36).substring(7).split('').join('.')
```

Comment:
Does it makes sense to have this *random token generator* in a function ? (since it being used at least twice here)

Reply:
Probably, if we start having more than just two. I'm find with the duplication for now.

Reply:
Why do you need two separate random strings generated? Seems that you can generate it once and use in both action types.

Reply:
Yes, but that seems like splitting hairs. It doesn't really matter.

Reply:
Ok, fair enough. There's also https://github.com/reactjs/redux/blob/master/src/combineReducers.js#L72. Does it make any sense to put all of them in the same place?

Reply:
I'm with @timdorr here.  When applying the D.R.Y. principle, it is critical to distinguish between essential duplication and coincidental duplication, because trying to "DRY-up" the latter kind can increase code complexity.  In cases where it is not clear what kind of duplication it is, one best practice is to permit the duplication until it causes actual pain.  This seems to be what @timdorr is doing.

Reply:
@quinned : highly relevant here: https://twitter.com/acemarke/status/901329101088215044 :)